src/section_handlers/document_tracking.py

```diff
@@ -0,0 +1,154 @@
+import logging
+import re
+import sys
+
+from typing import Tuple
+from src.common.common import SectionHandler
+from utils import get_utc_timestamp
+from operator import itemgetter
+from contextlib import suppress
+
+
+class DocumentTracking(SectionHandler):
+    def __init__(self, cvrf2csaf_name, cvrf2csaf_version):
+        super().__init__()
+        self.cvrf2csaf_name = cvrf2csaf_name
+        self.cvrf2csaf_version = cvrf2csaf_version
+
+    def _process_mandatory_elements(self, root_element):
+        self.csaf['id'] = root_element.Identification.ID.text
+        self.csaf['current_release_date'] = root_element.CurrentReleaseDate.text
+        self.csaf['initial_release_date'] = root_element.InitialReleaseDate.text
+        self.csaf['status'] = root_element.Status.text
+
+        revision_history, version = self._process_revision_history(root_element)
```

Comment:
Nit: Maybe, we should think of another name as it also returns the version? E.g. `_process_revision_history_and_version` But I'm +0 for that...

@sthagen: What do you think?

Reply:
@tschmidtb51 yes, process is a late and weak binding word ... preprocess even worse ... maybe split the functionality in two methods:
1. `revision_history(self, root_element) -> list` 
2.  `latest_version(self, hist: list) -> str`
and process the returned `history` (from calling 1.) in the small method (2.) that returns the `version` - so make it two calls for two things we harvest.